app/routes/dashboard.apps.$appId.notifications/components/NotificationsAlertForm/NotificationsAlertForm.tsx

```diff
@@ -65,67 +28,83 @@ export default function NotificationsAlertForm({
 }: NotificationsAlertFormProps) {
   const navigation = useNavigation()
   const { notificationSettings } = endpoint
+  const [changeHappened, setChangeHappened] = useState(false)
+  const [resetKey, setResetKey] = useState(135813)
 
   return (
-    <Form method="put">
-      <section className="pokt-network-notifications-activate-alerts">
+    <Form key={resetKey} method="put">
+      <Box w="100%">
         <Card>
-          <div className="pokt-card-header">
-            <h3>Activate Alerts</h3>
-          </div>
-          <p>
-            Set up usage alerts to be warned when you are approaching your relay limits.
-            We will send an email when your usage crosses the thresholds specified below.
-          </p>
-          <div className="pokt-network-notifications-alerts">
+          <Flex justify="space-between" w="100%" wrap="wrap">
+            <Title order={4}>Email Notifications</Title>
+
+            {changeHappened && (
+              <Flex gap="xs">
+                <Button
+                  disabled={
+                    navigation.state === "loading" || navigation.state === "submitting"
+                  }
+                  variant="outline"
+                  onClick={() => setResetKey((prev) => prev + 1)}
```

Comment:
Why do we need this approach? Can we use `<input type="reset">` instead?

Reply:
Got it... Just thinking out loud here, we can do something more simple if possible with Remix form.. like using a ref a then calling f`ormRef.reset()` if it exists? 

if not I'd just keep using the key work-around but I'd use `math.random()` instead of the predefined number